README.md

```diff
@@ -28,6 +28,7 @@ After cloning this repository, you can install the required dependencies
 
 ```
 pip install -r requirements.txt
+npm install --prefix ./fhir2dataset/metadata/node_modules 
```

Comment:
I think npm install must be done in the metadata folder, where the package.json file is
npm install --prefix ./fhir2dataset/metadata

Reply:
Absolutely! It's changed